rpg.py

```diff
@@ -0,0 +1,179 @@
+from dataclasses import dataclass, field
+import time
+from queue import PriorityQueue
+from typing import List, Tuple
+import random
+
+# Name input
```

Comment:
Ok. We've be over this before. Do this input name handling under `if __name__ == '__main__':